client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +365,16 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Get list of TransactionInfo for the Stream having status COMMITTED/ABORTED from most recent batch.
+     * TransactionInfo contains unique transactionId as well as status of transactions.
```

Comment:
same as above.... I see `TransactionInfo` has stream as well

Reply:
Yes, updated the detail.